commerce_shipping.module

```diff
@@ -62,6 +64,25 @@ function commerce_shipping_field_widget_form_alter(&$element, FormStateInterface
   }
 }
 
+/**
+ * Implements hook_entity_operation().
+ */
+function commerce_shipping_entity_operation(EntityInterface $entity) {
+  if ($entity->getEntityTypeId() === 'commerce_order') {
+    if (\Drupal::currentUser()->hasPermission('administer commerce_shipment')) {
+      $operations = [];
+      $operations['shipments'] = [
+        'title' => t('Shipments'),
+        'url' => Url::fromRoute('entity.commerce_shipment.collection', [
+          'commerce_order' => $entity->id(),
+        ]),
+        'weight' => 50,
+      ];
+      return $operations;
+    }
+  }
+}
+
```

Comment:
**CHANGE REQUEST**

*Readability*

You know, the usual readability comment: avoid unnecessary indentation levels:

```
if ($entity->getEntityTypeId() !== 'commerce_order') {
  return;
}

if (!\Drupal::currentUser()->hasPermission('administer commerce_shipment')) {
  return;
}

// Code here.
```